### DISCUSSION TITLE
network usage graph: progressive/logarithmic results ?

### DISCUSSION THREAD

**Discussion by lafleurdeboum** [ClementTsang/bottom#1108] [Q&A] [answered]
Would it be possible to have progressive scales on some results ? I think it would prove useful with network usage, with the lower third of the graph for bits, the middle third for kilobits and the upper third for megabits.

Also, when the network widget is very small, the network bandwidth unit (along with the zero, very clever) disappears all too soon IMHO, it being the most significant information of that widget. It's the case with the default layout on a 80x24 terminal, which I still use. In fact I switched it to 80x26 recently to accommodate _that_ aspect, but I hate it.

I've been using `btm` for years now and it's the only thing that I figured could be better :-)

**Comment by ClementTsang** [comment, marked as answer]
Just to check, does `--network_use_log` help for your first use case?

As for the second, I can look and see if that can be made more lenient regarding hiding/drawing order.

**Comment by lafleurdeboum** [comment]
A_w_e_s_o_m_e

I hadn't found that option looking into the config - guess this discussion will help people as lazy as me

Le April 20, 2023 6:14:02 PM UTC, Clement Tsang ***@***.***> a écrit :
